tests/zfs-tests/cmd/libzfs_input_check/libzfs_input_check.c

```diff
@@ -103,6 +104,10 @@ static unsigned ioc_skip[] = {
 	ZFS_IOC_EVENTS_NEXT,
 	ZFS_IOC_EVENTS_CLEAR,
 	ZFS_IOC_EVENTS_SEEK,
+
+	ZFS_IOC_NEXTBOOT,
```

Comment:
Can you add a one line comment above the Linux and and FreeBSD ioctls which are skipped.